dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp

```diff
@@ -82,14 +80,13 @@ void MinMaxIndex::addPack(const IColumn & column, const ColumnVector<UInt8> * de
         }
     }
 
-    const IColumn & updated_column = *column_ptr;
-    auto [min_index, max_index] = details::minmax(updated_column, del_mark, 0, updated_column.size());
+    auto [min_index, max_index] = details::minmax(column, del_mark, 0, column.size());
```

Comment:
What's the `min_index` of we call `details::minmax` with a column `[NULL, 1, 2, NULL]`? Is `min_index` point to the `NULL` or point to the `1`?

Reply:
https://github.com/pingcap/tiflash/blob/164eda5296fe0714ba3e7a30f0ffd7a562b378be/dbms/src/Storages/DeltaMerge/Index/MinMaxIndex.cpp#L51 The `null_direction_hint` is `-1`, it means that NULLs are considered as least than everything other, so the `min_index` will point to the `NULL`.